demos/list.html

```diff
@@ -1200,6 +1329,35 @@ <h3>Example - Interactive List (Dark Theme)</h3>
             evt.preventDefault();
           });
         }
+
+        function clickHandler(evt, rippleFoundation, li) {
```

Comment:
Why do you need all of this code? Since you're instantiating a ripple on each li anyway, you shouldn't need to activate it manually.

Shouldn't the following be enough to work?

```js
          var checkbox = li.querySelector('.mdc-checkbox__native-control');
          if(evt.target === li) {
            checkbox.checked = !checkbox.checked;
          }
```

And then you don't even need `rippleFoundation`.